zebra-chain/src/sapling/commitment.rs

```diff
@@ -200,13 +241,22 @@ impl ValueCommitment {
     ///
     /// https://zips.z.cash/protocol/protocol.pdf#concretehomomorphiccommit
     #[allow(non_snake_case)]
-    pub fn new<T>(csprng: &mut T, value: Amount<NonNegative>) -> Self
+    pub fn randomized<T>(csprng: &mut T, value: Amount) -> Self
```

Comment:
What is the reason for the new function in the context of this PR ?

Reply:
This snuck over into this PR from the WIP over here: https://github.com/ZcashFoundation/zebra/pull/1100